run-admin-web-application.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Script to run the CoatRack web application (after it has been built using 'build.sh')
+
+# After startup, please use a web browser to access the application at http://localhost:8080
+
+# Please note: 
+# - running CoatRack requires some mail configuration parameters to be set
+# - the configuration values in this file are just dummy/fallback values for development builds
+
+COATRACK_JAR_FILE='spring-boot/admin/target/coatrack-admin-*.jar'
+
+
+if [ -f $COATRACK_JAR_FILE ]; then 
+
+	java -jar -Dygg.mail.sender.user="" -Dygg.mail.sender.password="" -Dygg.mail.server.url="" -Dygg.mail.server.port=0 $COATRACK_JAR_FILE
+
+else
+	echo "Error: CoatRack executable file '$COATRACK_JAR_FILE' could not be found."
```

Comment:
why not do a mvn package here and run?

Reply:
Agree, makes sense... we could simply call `build.sh` in this case

Reply:
@Monezz I updated the script: in case there is no JAR file, the application is now built first and then started